db/controllers/topicsController.js

```diff
@@ -0,0 +1,16 @@
+const { selectTopics } = require("../models/topics");
+
+exports.getHealthcheck = (req, res) => {
+  res.status(200).send({ msg: "All okay." });
+};
+
+exports.getTopics = (req, res, next) => {
+  selectTopics()
+    .then((topics) => {
+      if (!topics || topics.length === 0) {
+        return Promise.reject({ status: 404, message: "Topics not found." });
```

Comment:
There isnt a need for this code as there isnt a test thats failing for it. Please only add the code that responds to the tests you have. Not sure when this would be needed.